app/src/main/res/layout/component_sign_in.xml

```diff
@@ -8,7 +8,7 @@
     android:id="@+id/bookmark_folders_sign_in"
     style="@style/NeutralButton"
     android:text="@string/bookmark_sign_in_button"
-    android:visibility="gone"
```

Comment:
nit - we can remove this line entirely I think? AFAICT we're not messing with the visibility of this view dynamically anymore